docs/wiki/CL/SSZ.md

```diff
@@ -0,0 +1,823 @@
+# Simple Serialize (SSZ)
+
+## Overview
+
+Simple Serialize (SSZ) is a serialization and [Merkleization](/docs/wiki/CL/merkleization.md) scheme designed specifically for Ethereum's Beacon Chain. SSZ replaces the [RLP serialization](/docs/wiki/EL/RLP.md) used on the execution layer (EL) everywhere across the consensus layer (CL) except the [peer discovery protocol](https://github.com/ethereum/devp2p). Its development and adoption are aimed at enhancing the efficiency, security, and scalability of Ethereum's CL.
+
+This document is about SSZ Serialization. You can learn more about SSZ merkleization at the [merkleization wiki page](/docs/wiki/CL/merkleization.md).
+
+
+## SSZ Tools
+
+There are many tools available for SSZ. Here is a [full list](https://github.com/ethereum/consensus-specs/issues/2138) of SSZ tools. Below are some of the popular ones:
+
+- [py-ssz](https://github.com/ethereum/py-ssz)
+- [dafny](https://github.com/ConsenSys/eth2.0-dafny)
+- [Eth2.py](https://github.com/protolambda/remerkleable)
+- [fastssz](https://github.com/ferranbt/fastssz/)
+- [rust-ssz](https://github.com/ralexstokes/ssz-rs)
+
+## SSZ VS RLP Serialization
+
+| CRITERIA   | COMPACT | EXPRESSIVENESS | HASHING  | INDEXING |
+|------------|---------|----------------|----------|----------|
+| RLP        | Yes     | Flexible       | Possible | No       |
+| SSZ        | No      | Yes            | Yes      | Poor     |
+
+_Table: SSZ VS RLP Comparison by [Piper Merriam](https://twitter.com/pipermerriam)._
+
+1. **Expressiveness**:
```

Comment:
Normally we use `1.` for all the ordered lists items, and the markdown renderer will set the correct index. Makes it easier if you need to reorder something or add something in the middle, not needing to renumber everything.